worker/raft/metrics.go

```diff
@@ -17,7 +17,7 @@ func registerMetrics(registerer prometheus.Registerer) (prometheus.Collector, er
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
-	_, err = metrics.NewGlobal(metrics.DefaultConfig("jujumetrics"), sink)
+	_, err = metrics.NewGlobal(metrics.DefaultConfig("juju"), sink)
```

Comment:
I think this needs to be something other than "juju" (which is used for metrics in our code) - otherwise there's a possibility of the register failing because of name collisions between juju/juju metrics and hashicorp/raft ones.

Reply:
I checked, and they all say "juju_raft_..."

Reply:
If we find it a problem later, we can look at it then, but this makes all our metrics register nicely with the same prefix.

Reply:
Yeah, fair enough.